### ISSUE TITLE
send current detected gear over CAN

### ISSUE THREAD

**Issue by mck1117** [rusefi/rusefi#5101]
$title

**Comment by Benjamin-neo** [comment]
will be great if you can add "EGO correction" and "knock retard" over CAN. I am currently made a Dash over can and three info are missing.

**Comment by rusefillc** [comment]
@Benjamin-neo anything you can post on the forum about the dash?

**Comment by Benjamin-neo** [comment]
yes I will post when it will ready. At this moment, lot of things works. I must finish my ecu (m52bmw pnp) completely before finish dash to see the best usefull and discover future bug. I can only tell you will be release on 5 and 7 inch touch screen, some adjustable visual and sound alert on clt , rpm, fuel, boost. I think will post next month some pics.
